server/src/test/integration/drinksEndpointTest.js

```diff
@@ -0,0 +1,62 @@
+const app = require('../../server');
+const request = require('supertest');
+const expect = require('chai').expect;
+const dbAdapter = require('../../db/adapter');
+
+describe(`'drinks' route - api test`, function() {
+  // before the tests, prime the database with test tables/data
+  before(async function() {
+    const tables = await dbAdapter.r.tableList();
+    const drinkTestObjects = [
+      {
+        ingredients: ['whiskey', 'sugar', 'bitters'],
+        name: 'Old Fashioned'
+      },
+      {
+        ingredients: ['gin', 'sugar', 'lemon juice', 'club soda'],
+        name: 'Tom Collins'
+      }
+    ];
+
+    if (tables.includes('drinks')) {
+      await dbAdapter.r.tableDrop('drinks');
+    }
+
+    await dbAdapter.r.tableCreate('drinks', { primaryKey: 'name' });
+    await dbAdapter.r.table('drinks').insert(drinkTestObjects);
+    return Promise.resolve();
+  });
+
+  // after the tests, drain the connection pool so the process exits properly
+  after(async function() {
+    await dbAdapter.r.getPoolMaster().drain();
+    return Promise.resolve();
+  });
+
+  // test a successful hit to get all drinks
```

Comment:
Personal pref but I feel like these comments are redundant with the descriptions in the `it` blocks

Reply:
Yeah I put those there in case I wanted to explain more because I didn't want to write very much in the `it` blocks. for these it wasn't really necessary though